front-end-notes/angular.md

```diff
@@ -0,0 +1,112 @@
+#AngularJS
+Angular implements a MVC design pattern, which stands for Model View Controller.
+
+###Model
+Model is strictly data. It does NOT depend on the controller or the view.
+
+###View
+The view is an interface to view and modify the data.
+
+###Controller
+Controllers are operations that can be performed on the data.

```

Comment:
A more rigorous definition of controllers might be helpful, particularly describing what is appropriate to put in a controller and what should be reserved for directives and services, etc.